src/elements/content-sidebar/versions/VersionsList.js

```diff
@@ -34,20 +35,36 @@ const VersionsList = ({ isLoading, versions, ...rest }: Props) => {
 
     return (
         <ul className="bcs-VersionsList">
-            {versions.map((version, index) => (
-                <li className="bcs-VersionsList-item" key={version.id}>
-                    <Route
-                        render={({ match }) => (
-                            <VersionsItem
-                                isCurrent={index === 0}
-                                isSelected={match.params.versionId === version.id}
-                                version={version}
-                                {...rest}
-                            />
+            {versions.map((version, index) => {
+                const isCurrent = index === 0;
+                const previousVersion = versions[Math.max(0, index - 1)];
+                const previousGroup = getGroup(previousVersion);
+                const newGroup = getGroup(version);
+                const hasGroup = isCurrent || previousGroup !== newGroup;
+
+                return (
+                    <React.Fragment key={version.id}>
+                        {hasGroup && (
+                            <li className="bcs-VersionsList-heading">
```

Comment:
Is there another semantic way to do subheaders in a list? I'm wondering how screen readers might read this

Reply:
Good question. This is how our dropdown menu works, so I followed the convention here as well. I'll take a quick look online to see what I find.